web/react/components/invite_member_modal.jsx

```diff
@@ -89,146 +90,149 @@ module.exports = React.createClass({
+            idCount: 0,
+            emailErrors: {},
+            firstNameErrors: {},
+            lastNameErrors: {}
         });
     },
     removeInviteFields: function(index) {
-        var count = this.state.id_count;
-        var invite_ids = this.state.invite_ids;
-        var i = invite_ids.indexOf(index);
-        if (i > -1) invite_ids.splice(i, 1);
-        if (!invite_ids.length) invite_ids.push(++count);
-        this.setState({ invite_ids: invite_ids, id_count: count });
+        var count = this.state.idCount;
+        var inviteIds = this.state.inviteIds;
+        var i = inviteIds.indexOf(index);
+        if (i > -1) {
+            inviteIds.splice(i, 1);
+        }
+        if (!inviteIds.length) {
+            inviteIds.push(++count);
+        }
+        this.setState({inviteIds: inviteIds, idCount: count});
     },
     getInitialState: function() {
         return {
-            invite_ids: [0],
-            id_count: 0,
-            email_errors: {},
-            first_name_errors: {},
-            last_name_errors: {}
+            inviteIds: [0],
+            idCount: 0,
+            emailErrors: {},
+            firstNameErrors: {},
+            lastNameErrors: {}
         };
     },
     render: function() {
-        var currentUser = UserStore.getCurrentUser()
+        var currentUser = UserStore.getCurrentUser();
 
         if (currentUser != null) {
-            var invite_sections = [];
-            var invite_ids = this.state.invite_ids;
-            var self = this;
-            for (var i = 0; i < invite_ids.length; i++) {
-                var index = invite_ids[i];
-                var email_error = this.state.email_errors[index] ? <label className='control-label'>{ this.state.email_errors[index] }</label> : null;
-                var first_name_error = this.state.first_name_errors[index] ? <label className='control-label'>{ this.state.first_name_errors[index] }</label> : null;
-                var last_name_error = this.state.last_name_errors[index] ? <label className='control-label'>{ this.state.last_name_errors[index] }</label> : null;
-
-                invite_sections[index] = (
-                    <div key={"key" + index}>
+            var inviteSections = [];
+            var inviteIds = this.state.inviteIds;
+            for (var i = 0; i < inviteIds.length; i++) {
+                var index = inviteIds[i];
+                var emailError = this.state.emailErrors[index] ? <label className='control-label'>{this.state.emailErrors[index]}</label> : null;
+                var firstNameError = this.state.firstNameErrors[index] ? <label className='control-label'>{this.state.firstNameErrors[index]}</label> : null;
+                var lastNameError = this.state.lastNameErrors[index] ? <label className='control-label'>{this.state.lastNameErrors[index]}</label> : null;
+
+                inviteSections[index] = (
+                    <div key={'key' + index}>
+                    {index ?
```

Comment:
These ternaries will need to be removed to conform to the style guide. Probably should do that now.